### DISCUSSION TITLE
Can't add multi cursor with <C-n> on MacOS

### DISCUSSION THREAD

**Discussion by ilovecomputers** [JetBrains/ideavim#577] [Q&A] [answered]
I'm using Webstorm on MacOS 13. I have `set multiple-cursors` added to my .ideavimrc. I removed the keymap for \<C-n> in the settings. However, \<C-n> continues to move the cursor down instead of adding a cursor.

**Comment by citizenmatt** [comment, marked as answer]
Can you share your `.ideavimrc`? The default mappings for the multiple cursors extension are surprisingly not the same as the Vim plugin it emulates, e.g. it uses `<A-n>`. There is an issue for this, but it hasn't been implemented yet: [VIM-2178](https://youtrack.jetbrains.com/issue/VIM-2178).

That issue also includes a comment that contains an `.ideavimrc` snippet to remap to give you correct key bindings.

**Reply by ilovecomputers** [reply]
I haven't remapped anything, so the snippet in the linked issue fixed it for me, thanks!
